portafly/README.md

```diff
@@ -1,6 +1,31 @@
 # PortaFly
 A React SPA crafted with the ambitious purpose of replacing Porta UI, currently residing inside the Rails App.
 
+## Getting up and running
+1. Clone the `porta` repository to your local machine
+2. Create a `.env` file:
+  * Create a new file in the `/portafly` root folder, and name it `.env`
+  * Paste `REACT_APP_API_HOST="https://multitenant-admin.preview01.3scale.net"` into it, and Save
+3. Create an Access token:
```

Comment:
This step is right (by now), but it's not the definitive way to authenticate, just a temporary solution. IDK if we should add this section, as this is going to change in the future...

Reply:
you wouldn't be able to access portafly without an access token right now, so I think it is a necessary step, although temporary. we will remove it when things change.